tests/baselines/reference/classStaticBlock2(target=es5).js

```diff
@@ -0,0 +1,40 @@
+//// [classStaticBlock2.ts]
+const a = 1;
+const b = 2;
+
+class C {
+    static {
+        const a = 11;
+
+        a;
+        b;
+    }
+
+    static {
+        const a = 11;
+
+        a;
+        b;
+    }
+}
+
+
+//// [classStaticBlock2.js]
+var _C__, _C__1;
+var a = 1;
+var b = 2;
+var C = /** @class */ (function () {
+    function C() {
+    }
+    return C;
+}());
+_C__ = { value: (function () {
+        var a_1 = 11;

```

Comment:
Another case where there should be no collision because we're in a new function scope, this should be `a` not `a_1`.